app/Models/AccessPoint.php

```diff
@@ -2,8 +2,40 @@
 
 namespace App\Models;
 
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use LibreNMS\Interfaces\Models\Keyable;
+
 class AccessPoint extends DeviceRelatedModel
```

Comment:
Missing `implements Keyable` on the class declaration

Reply:
Added